packages/rs-dpp/src/state_transition/state_transitions/identity/identity_credit_withdrawal_transition/v0/mod.rs

```diff
@@ -8,17 +8,24 @@ mod value_conversion;
 mod version;
 
 use bincode::{Decode, Encode};
+use dashcore::transaction::special_transaction::asset_unlock::qualified_asset_unlock::ASSET_UNLOCK_TX_SIZE;
 use platform_serialization_derive::PlatformSignable;
 use platform_value::BinaryData;
 use serde::{Deserialize, Serialize};
 
 use crate::{
     identity::{core_script::CoreScript, KeyID},
     prelude::{Identifier, Revision},
+    withdrawal::Pooling,
     ProtocolError,
 };
 
-use crate::withdrawal::Pooling;
+/// Minimal core per byte. Must be a fibonacci number
+pub const MIN_CORE_FEE_PER_BYTE: u32 = 5;
```

Comment:
I believe this should start at 1, this should also be versioned.

Reply:
Answered here https://github.com/dashpay/platform/pull/1601#discussion_r1479160085

Reply:
This needs a todo then, I don't know why it's in multiple places either